README.md

```diff
@@ -42,7 +42,7 @@ These differences are:
 In order to build Wire for iOS locally, it is necessary to install and setup the following tools on the local machine:
 
 - macOS 13.5 (Ventura) or later
```

Comment:
From the [Xcode release notes](https://developer.apple.com/documentation/xcode-release-notes/xcode-15_3-release-notes):
"Xcode 15.3 requires a Mac running macOS Sonoma 14 or later."

Maybe we can omit this line.